doc/manuals/user/index.md

```diff
@@ -0,0 +1,118 @@
+# Usage Overview
+
+First you need to have Quantumleap runnning (of course). Refer to the [Installation Manual](../admin/index.md) for instructions.
+
+Then, you need to connect Orion Context Broker to Quantumleap through a subscription for each Entity Type whose historical data you are interested in. Historical data for each entity type will be persisted as long as the subscription is active.
+
+The flow is therefore the following
+
+- Create an Orion subscription for each entity type of your interest once
+- Insert and update your data to Orion as usual
+- Done, your historical data will be persisted in Quantumleap's database.
+
+Let's have a look in more detail of each step.
+
+
+# Orion Subscription
+
+As stated before, the link between Orion and Quantumleap is established through a subscription you need to create. It is therefore important that you understand well how the NGSIv2 Subscription mechanism works. This is carefully explained in the corresponding section of [Orion docs](https://fiware-orion.readthedocs.io/en/master/user/walkthrough_apiv2/index.html#subscriptions).
+
+Here's an example of the payload of the subscription you need to create in Orion to establish the link Orion-Quantumleap.
+
+    {
+        "description": "Test subscription",
+        "subject": {
+            "entities": [
+            {
+                "idPattern": ".*",
+                "type": "Room"
+            }
+            ],
+            "condition": {
+                "attrs": [  # Here the list of attrs whose change will trigger notifications. Empty array means track any change.
+                "temperature",
```

Comment:
A minimal typo: "temperature" shouldn't have comma in this example

Reply:
Thanks @netzahdzc!